doc/examples/README.md

```diff
@@ -2,6 +2,8 @@
 
 This folder contains a collection of more specific examples using flame, specially to showcase some more advanced functionality.
 
-To run this examples after cloning, first run `flutter create .` inside to generate the necessary files, as per instructed by the other flutter commands.
+To run this examples after cloning, first run `flutter create .` inside the desired example folder to generate the necessary files, as per instructed by the other flutter commands.
 
-After that, you should be able to `flutter run` just fine.
+After that, run `flutter pub get` to get all dependencies as defined in the pubspec.yml file.
+
+Finaly, you should be able to `flutter run` just fine.
```

Comment:
typo: `Finally`

Reply:
Let me fix that! I also found another problem with one of the examples which I fixed